dask/dataframe/tests/test_dataframe.py

```diff
@@ -240,27 +240,6 @@ def test_index_names():
     assert ddf.index.compute().name == "x"
 
 
-@pytest.mark.parametrize(
-    "npartitions",
-    [
-        1,
-        pytest.param(
-            2,
-            marks=pytest.mark.xfail(
-                not dd._compat.PANDAS_GT_110, reason="Fixed upstream."
-            ),
-        ),
-    ],
-)
-def test_timezone_freq(npartitions):
```

Comment:
Why is this being removed?

Reply:
The `freq` argument is being removed from timestamps entirely. Consider the pure pandas case:

```python
In [1]: import pandas as pd
   ...: idx = pd.date_range("20130101", periods=10, tz="US/Eastern")
   ...: idx[0].freq
<ipython-input-1-a14ec23a1620>:3: FutureWarning: Timestamp.freq is deprecated and will be removed in a future version
  idx[0].freq
Out[1]: <Day>
```

Reply:
Ah, I see. That makes sense. Can we keep this test around for older versions of pandas then?

Reply:
We can yeah, just wasn't sure if it is something that we care about checking since it is going away in future pandas. But I am fine either way.

Reply:
I think it'd be good to keep around, especially since we already have the test and still support older versions of pandas. We can use our pandas version compat variables

https://github.com/dask/dask/blob/7af542afaeba9eea46cea653c51a02a2c3376962/dask/dataframe/_compat.py#L7-L13

which will make it more clear in the future when we can drop this test altogether